cmd/switcher/switcher.go

```diff
@@ -240,13 +245,30 @@ func init() {
 		true,
 		"run hooks right away. Do not respect the hooks execution configuration.")
 
+	versionCmd := &cobra.Command{
```

Comment:
Can you also add the `switch version `(& `switch -v`)  command in the [switch.sh](https://github.com/danielfoehrKn/kubeswitch/blob/b6bb0b6449659d72cfc7a71ab2c6ba931d194dd3/hack/switch/switch.sh) script that calls the switcher binary?

Reply:
done 
![image](https://user-images.githubusercontent.com/42234376/111104003-a11dd500-858a-11eb-8769-6751bc0368b1.png)

Reply:
Maybe a misunderstanding. 

This is not the switch**.sh** script, but the **switcher binary**. 
Users use the **switch.sh** script directly, not the switcher. 
So only adding it to the switcher binary is not enough.

Please add the version command also [here](https://github.com/danielfoehrKn/kubeswitch/blob/b6bb0b6449659d72cfc7a71ab2c6ba931d194dd3/hack/switch/switch.sh)
so that you can call `switch version`.